tests/integration-tests/test_runner.py

```diff
@@ -248,6 +248,15 @@ def _is_file(value):
     return value
 
 
+def _join_with_not(args):
```

Comment:
A docstring explaining why we need this would be useful.

Reply:
done